### PR TITLE
display some placeholder content with twitter bootstrap

### PR DESCRIPTION
**Pull request by jdimatteo**
Please login to https://wegrowwealth.org/ and review the UX

### CODE REVIEW

**Comment by anfedorov** [comment]
sorry I missed this — you didn't add me as a reviewer so I didn't get an email :-)

**Comment by anfedorov** [comment]
same as last time? 
image


done.

**Comment by jdimatteo** [comment]
Thanks.  I didn't add you as a reviewer last time -- I just referenced you
in a comment.

File: .eb_saved_configs/WeGrowWealth-env-sc.cfg.yml [hunk 1]
```diff
@@ -30,6 +30,10 @@ OptionSettings:
     PreferredStartTime: TUE:07:25
   aws:elasticbeanstalk:managedactions:platformupdate:
     UpdateLevel: minor
+  aws:elb:policies:
+    Stickiness Policy: true
```
[the remark this answers is not in the document]
Comment (jdimatteo) [reply .eb_saved_configs/WeGrowWealth-env-sc.cfg.yml]: fyi, stickiness is important when doing zero downtime upgrades so people aren't swapping randomly between different versions by the loadbalancer.  it is also helpful if we setup local caching

**Pull request merged**